apps/nextjs-website/react-components/components/common/Common.tsx

```diff
@@ -5,9 +5,11 @@ import { CtaButtonProps } from '../../types/common/Common.types';
 export const CtaButtons = ({
   ctaButtons,
   theme,
```

Comment:
```suggestion
  theme = 'light',
```
Should we set a default value to theme as well to avoid having to explicitly set it for components like the PreHeader which are light-mode only?